src/main/java/emissary/core/IBaseDataObjectXmlCodecs.java

```diff
@@ -28,10 +29,39 @@
 import java.util.stream.Stream;
 import javax.xml.XMLConstants;
 
+import static emissary.core.constants.IbdoMethodNames.ADD_ALTERNATE_VIEW;
+import static emissary.core.constants.IbdoMethodNames.ADD_PROCESSING_ERROR;
+import static emissary.core.constants.IbdoMethodNames.SET_BIRTH_ORDER;
+import static emissary.core.constants.IbdoMethodNames.SET_BROKEN;
+import static emissary.core.constants.IbdoMethodNames.SET_CLASSIFICATION;
+import static emissary.core.constants.IbdoMethodNames.SET_CURRENT_FORM;
+import static emissary.core.constants.IbdoMethodNames.SET_DATA;
+import static emissary.core.constants.IbdoMethodNames.SET_FILENAME;
+import static emissary.core.constants.IbdoMethodNames.SET_FONT_ENCODING;
+import static emissary.core.constants.IbdoMethodNames.SET_FOOTER;
+import static emissary.core.constants.IbdoMethodNames.SET_HEADER;
+import static emissary.core.constants.IbdoMethodNames.SET_HEADER_ENCODING;
+import static emissary.core.constants.IbdoMethodNames.SET_ID;
+import static emissary.core.constants.IbdoMethodNames.SET_NUM_CHILDREN;
+import static emissary.core.constants.IbdoMethodNames.SET_NUM_SIBLINGS;
+import static emissary.core.constants.IbdoMethodNames.SET_OUTPUTABLE;
+import static emissary.core.constants.IbdoMethodNames.SET_PARAMETER;
+import static emissary.core.constants.IbdoMethodNames.SET_PRIORITY;
+import static emissary.core.constants.IbdoMethodNames.SET_TRANSACTION_ID;
+import static emissary.core.constants.IbdoMethodNames.SET_WORK_BUNDLE_ID;
 import static emissary.core.constants.IbdoXmlElementNames.BASE64_ATTRIBUTE_NAME;
```

Comment:
While we're cleaning these up, let's make this move this constant into the IBaseDataObjectXmlCodecs class itself and rename it as `BASE64`.  Its value is "base64", it's only used within this class, and it's neither an element name nor an attribute name (it's a value we sometimes assign to the "encoding" attribute).  No Javadoc comment needed for it, it's self-documenting.